AnalyticsKit.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name         = "AnalyticsKit"
-  s.version      = "2.0.0"
+  s.version      = "2.1.0"
```

Comment:
I know there might be some breaking changes but I tried to keep things backwards compatible and if anyone finds a problem I can fix that so we don't need to bump to 3.0